.github/workflows/inspektor-gadget.yml

```diff
@@ -716,40 +712,35 @@ jobs:
     strategy:
       matrix:
         type: [default, core]
-        driver: [none, docker]
+        profile: [docker, containerd, cri-o, multi-node]
```

Comment:
What's special about `multi-node` to be a different profile? Is it possible to always test "multi-node"?

Reply:
Good question. I think it helps to identify if something is specifically broken for multi node installation. Can't think of exact use-case so sounds fine to me if we choose to run all profiles with multi-node. (need to be careful with the resources on GH runner)

cc @blanquicet since we requested this as part of #968

Reply:
Absolutely, all the runtime must run on a multi-node cluster.

> need to be careful with the resources on GH runner
>
We have been already running multi-node for docker runtime for a while, I don't expect others runtime to consume a lot more.

Reply:
Cool. I will update the PR to run all runtime with multi-node setup. This also covers single node scenario so we should be fine there as well!